tests/test_hydrate.py

```diff
@@ -0,0 +1,86 @@
+import re
+import json
+import pytest
+
+from rdflib import Graph, URIRef
+from store.metadata.hydrate import _hydrate_graph_from_sub_graphs
+from pyld import jsonld
+
+from src.store.metadata import constants
+from src.store.metadata.oxigraph.store import OxigraphMetadataStore, _get_single_graph_for_field
+
+def _populate_partial_graph_for_test():
```

Comment:
this is clever but you can make this a lot easier on yourself.

1.) just run the api and log out (or dump to disk) the results of [this line](https://github.com/GSS-Cogs/idpd-api-poc/blob/44270c961dab0b013358bb6efef631bce31c35e0/src/store/metadata/oxigraph/store.py#L99) and make it a pytest json/dict fixture along the same lines as the other (closer to end product) json fixtures we have. Once you've got that starting point...

2.) use `_get_single_graph_for_field` to get the main graph.

3.) the whole list of graphs should be accessible in that fixture under the `@graph` key.

4.) pass those two things to your function and do your assertions as you already are.

It's not very subtle but it'll be easy to understand which is the main thing really. As long as you've got that fixture as a starting point you can probably just put steps 2 and 3 in the actual test.